packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -475,7 +494,7 @@ export default {
     // eslint-disable-next-line complexity
     extensions () {
       // These are the default extensions needed just for plain text.
-      const extensions = [Document, Text, History, HardBreak];
+      const extensions = [Document, Text, TextStyle, History, HardBreak];
```

Comment:
You forgot to remove this on vue 2. Looks good on vue 3

Reply:
done and resolved pnpm lock file